docs/interpreter/spark.md

```diff
@@ -343,6 +343,29 @@ myScalaDataFrame = DataFrame(z.get("myScalaDataFrame"), sqlContext)
   </div>
 </div>
 
+#### Alternative Object Retrieval Mechanism
+In addition to the programmatic access using `z.get()` shown above, 
+some interpreters also support inline interpolation of such objects
+into command lines by using substitution patterns of the form `{object-key}`.
+The following example shows how the value of an object `put` into `ZeppelinContext` 
+by the Scala interpreter is spliced into a command of the SQL interpreter.
+
+__In Scala interpreter:__
```

Comment:
you should still do `{% highlight scala %}` for proper highlighting, just don't use `div`